src/subcommand/stats_main.cpp

```diff
@@ -141,7 +141,14 @@ int main_stats(int argc, char** argv) {
 			utils::handle_gfa_odgi_input(infile, "stats", args::get(progress), num_threads, graph);
         }
     }
-    ///graph.display();
+
+    /// we need to check in advance if the graph is compacted
```

Comment:
The compacted IDs are needed only where we use the `position_map` vector, that is when `mean_links_length` and/or `sum_of_path_node_distances` are computed.

Reply:
We need to have this check if we have `-y, --yaml`.

Reply:
Let me push something.

Reply:
Done.